docs/tcpAgent.md

```diff
@@ -0,0 +1,140 @@
+# Launching inbound TCP agents
+Jenkins provides a number of ways of connecting remote agents.
+Two of the most popular are SSH agents and inbound TCP agents.
```

Comment:
```suggestion
Two of the most popular are outbound SSH agents and inbound TCP agents.
```

Or do we also have inbound SSH agents? Since Jenkins has some sort of ssh-server plugin.